internal/main.go

```diff
@@ -33,8 +35,26 @@ func RunTests(opts RunTestsOpts) {
 	// Exclude by package prefix
 	testPkgs := allPkgs
 	if len(opts.Excludes) > 0 {
-		testPkgsStr := shCmd("grep", shArgs{"-Ev", getExcludePattern(opts.Excludes)}, allPkgsStr)
-		testPkgs = splitLines(testPkgsStr)
+		testPkgs = []string{}
+		excludeRegexs := make([]*regexp.Regexp, 0, len(opts.Excludes))
+		for _, exclude := range opts.Excludes {
+			regex, err := regexp.Compile("^" + exclude + "$")
+			if err != nil {
+				log.Fatalf("cannot compile regex %q: %+v", exclude, regex)
+			}
+			excludeRegexs = append(excludeRegexs, regex)
+		}
+		for _, pkg := range allPkgs {
+			isIncluded := true
+			for _, regex := range excludeRegexs {
```

Comment:
Wouldn't it be more performant to make a single regex and thus not have the nested loops.
Thinking `^first/exclude|^second/exclude|^etc/etc`? 
It also means a single regex comparison per tested package.

Reply:
I'm just not sure if it doesn't have some edge-case, like if the regexes are not properly formed and the | will somehow do some nonsense

Reply:
I think the performance difference will be negligent (especially we spend most time on waiting on `go test` later anyway)